Makefile

```
@@ -3,39 +3,64 @@ SCRIPTS_BASE          ?= $(ROOT_DIR)/scripts
 GOLANG_CI_YAML_PATH   ?= ${ROOT_DIR}/golang-ci.yaml
 GOLANG_CI_ARGS        ?= --allow-parallel-runners --timeout=5m --config=${GOLANG_CI_YAML_PATH}
 
+##
+# Console Colors
+##
+GREEN  := $(shell printf "\033[0;32m")
+YELLOW := $(shell printf "\033[0;33m")
+WHITE  := $(shell printf "\033[0;37m")
+CYAN   := $(shell printf "\033[0;36m")
+RESET  := $(shell printf "\033[0m")
+
+
+##
+# Targets
+##
+.PHONY: help
+help: ## show this help.
```

I would keep the comments consistent
```suggestion
help: ## Show this help
```